app/common/Interceptor.js

```diff
@@ -0,0 +1,9 @@
+angular.module('batApp')
+ .factory('Interceptor', function ($localStorage) {
+   return {
+     request: function (config) {
+       config.url = config.url + '?accessToken=' + $localStorage.token
```

Comment:
if $localStorage.token have some value then access token will be added to configurl